packages/logger/src/lib/logger.ts

```diff
@@ -397,49 +408,82 @@ export class LogManager {
 
   // if a logger is given an id it will persist logs under its logger instance
   public get(category: string, id?: string): Logger {
-    let instance = this._loggers.get(`${category}${id}`);
-    if (instance !== undefined) {
+    let instance = this._loggers.get(category);
+    if (!instance && !id) {
+      this._loggers.set(
+        category,
+        Logger.createLogger(category, this._level ?? LogLevel.INFO, '', true)
+      );
+
+      instance = this._loggers.get(category) as Logger;
+      instance.Config = this._config;
       return instance;
     }
 
-    const logger = Logger.createLogger(
+    this._loggers.set(
       category,
-      this._level ?? LogLevel.INFO,
-      id ?? ''
+      Logger.createLogger(category, this._level ?? LogLevel.INFO, '', true)
```

Comment:
Repeated calls to `.get('sameCategory', 'someId')` will create a new logger each time and save those over and over to this set at `category`. Shouldn't we have a previous step where trying to get and return?

As I understand, this will recreate the LogManager default logger each time it has any log requested, even childs. This seems to be the `else` part of the `if` right below

Reply:
wow yes this is totally true, I guarded the creation of the logger in an `else` case so now it only creates a new instance when required. Good catch 😄

Reply:
@FedericoAmura 
I cleaned this up